frontend/src/Components/ContentComponent.module.scss

```diff
@@ -88,6 +88,14 @@
   }
 }
 
+.overlayCloseButton {
```

Comment:
sorry for late comment, I think it would be nice to have a shadow for clear vision on a light background.
![image](https://github.com/spaceshelter/orbitar/assets/230060/488e8e36-95f9-406d-9083-504260a014b3)
![image](https://github.com/spaceshelter/orbitar/assets/230060/f02c9868-6085-464e-85f0-b73982b804a8)